scripts/monitoring/ops-metric-client.py

```diff
@@ -64,6 +64,21 @@ def run(self):
 
         self.metric_sender.send_metrics()
 
+    @staticmethod
+    def adjust_type(val, metric_type):
+        """
+        :param val: string representing metric value
+        :param metric_type: string or numeric
+        :return: value converted to float if metric_type isn't string
+        """
+        if metric_type == 'string':
+            return val
+
+        try:
+            return eval(val)
```

Comment:
So it seems.
I could do the same with `json.loads` as we are trying to imitate it's behaviour